### PR TITLE
Fixes issue #43 and issue #44

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: nautobot_ssot_aristacv/diffsync/cvutils.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def connect():
         call_creds = grpc.access_token_call_credentials(response.json()["sessionId"])
     # Set up credentials for CVaaS using supplied token.
     else:
-        cvp_url = "www.arista.io:443"
+        cvp_url = PLUGIN_SETTINGS["cvaas_url"]
```
Comment (chipn) [inline nautobot_ssot_aristacv/diffsync/cvutils.py:43]: Should we use `.get` here and keep the original URL as default?

File: nautobot_ssot_aristacv/diffsync/fromcv/cloudvision.py [hunk 2]
```diff
@@ -15,25 +16,38 @@ class CloudVision(DiffSync):
...
             dev_tags = [tag for tag in cvutils.get_device_tags(device_id=dev["device_id"]) if tag in system_tags]
+
             # Check if topology_type tag exists
             list_of_tag_names = [value["label"] for value in dev_tags]
             if "topology_type" not in list_of_tag_names:
                 dev_tags.append({"label": "topology_type", "value": "-"})
 
             for tag in dev_tags:
-                if any(tag["label"] in string for string in ["hostname", "serialnumber"]):
+                if any(tag["label"] in string for string in ["hostname", "serialnumber", "Container"]):
```
Comment (chipn) [inline nautobot_ssot_aristacv/diffsync/fromcv/cloudvision.py:41]: I'm a bit confused by this one. Can't you just use `if tag["label"] in ["hostname", "serialnumber", "Container"]`?

**Review by chipn — CHANGES_REQUESTED** [review]

File: nautobot_ssot_aristacv/diffsync/cvutils.py (hunk 1 above)
Comment (qduk) [reply nautobot_ssot_aristacv/diffsync/cvutils.py:43]: Good idea! Updated.

File: nautobot_ssot_aristacv/diffsync/fromcv/cloudvision.py (hunk 2 above)
Comment (qduk) [reply nautobot_ssot_aristacv/diffsync/fromcv/cloudvision.py:41]: That does work. Not sure why I made it so complicated.

**Review by chipn — APPROVED** [review]
LGTM other than the CI failure (which per our Slack conversation should be resolved by rebase).

**Pull request merged**